### ISSUE TITLE
Training on own dataset for (128x128) --> (512x512) fails miserably !!

### ISSUE THREAD

**Issue by ManuBN786** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#154]
I was training on my own dataset for (128x128) --> (512x512)
I have attached the train logs : 
[train.log](https://github.com/user-attachments/files/17504451/train.log)
[val.log](https://github.com/user-attachments/files/17504453/val.log)

The tensorboard visualizations look like this:
![imageData](https://github.com/user-attachments/assets/34548abe-c392-4ec8-b575-c9c84c34243c)

And the losses & psnr aren't improving
![psnr](https://github.com/user-attachments/assets/1d3eea52-7b15-4cff-85e9-233118b118a5)
![loss](https://github.com/user-attachments/assets/27d88e11-72bf-4c99-b502-c696eb4f5069)

Please help me to fix it

**Comment by guangying121** [comment]
Hey, I met with the same problem, but, but my l_pix is normal like 0.05, my sr_img and PSNR are the same situation like yours. Did you solve it?
![image](https://github.com/user-attachments/assets/0df9412e-1d4c-4315-8ad8-ba43fa6b8734)

**Comment by mgl666** [comment]
I have the same question, how to fix it up?

**Comment by tA312** [comment]
Try to lower the learning rate? I have adjusted to batch_stize4, lr2e-5

**Comment by ManuBN786** [comment]
I could not fix it. So I gave up !!
